onedal/linear_model/tests/test_logistic_regression.py

```diff
@@ -64,3 +65,30 @@ def test_pickle(queue, dtype):
         result = model2.predict(X, queue=queue)
 
         assert_array_equal(expected, result)
+
+
+if daal_check_version((2024, "P", 600)):
```

Comment:
This condition mentions 2024.6.0 version while `onedal` and onther places - 2024.7.0. Are you sure you set correct versions?